### DISCUSSION TITLE
Wordpress Plugin to Connect Crypto Wallet?

### DISCUSSION THREAD

**Discussion by AMCarbonaro94** [phantom/docs#263] [Help & Support] [answered]
Hello,

I'm building a WordPress website with a generic WordPress freelancer. We are having a hard time finding a plugin which allows front end users to connect their crypto wallet ie.Phantom to create a user account. The wallet user account will be able to purchase my service with Solana plus other user rolls such as comment and like.

Do you know of any plugins? If not, how may I create this?

I attached a video reference using pump.fun's "connect wallet" function which does exactly what we need for our own site. 

Please help. 
Thank you
-Anthony C

https://github.com/user-attachments/assets/37184bf4-1838-4868-9178-1238ef2ac743

**Comment by adamdelphantom** [comment, marked as answer]
Hello @AMCarbonaro94,

I am aware of the following, though I have never used them myself.

https://x.com/SolPressPlugins
https://x.com/helio_pay
